projects/core/src/multi-location-initialized/multi-location-initialized.module.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { LOCATION_INITIALIZED } from '@angular/common';
+import { ModuleWithProviders, NgModule, inject } from '@angular/core';
+import { MULTI_LOCATION_INITIALIZED } from './multi-location-initialized-token';
```

Comment:
MULTI_LOCATION_INITIALIZED vs LOCATION_INITIALIZED_MULTI?

MULTI_LOCATION_INITIALIZED sounds to me like multiple locations are initialized.

Reply:
Agreed 👍